### PR TITLE
Issue #SB-15085 checking emailverified and phoneverified for existing…

### PR DESCRIPTION
**Pull request by Hari-stackroute**
… users

### CODE REVIEW

File: actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java [hunk 1]
```diff
@@ -679,6 +679,16 @@ private void setStateValidation(
   private Map<String, Boolean> updatedUserFlagsMap(
       Map<String, Object> userMap, Map<String, Object> userDbRecord) {
     Map<String, Boolean> userBooleanMap = new HashMap<>();
+    if(userDbRecord.get(JsonKey.EMAIL) != null && userDbRecord.get(JsonKey.EMAIL_VERIFIED) == null) {
```
Comment (manzarul) [inline actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java]: need to handle scenario for not null and value as true as well. same for below code

Comment (Hari-stackroute) [reply actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java]: yes taken care for existing users with true value.

File: actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java [hunk 2]
```diff
@@ -679,6 +679,18 @@ private void setStateValidation(
   private Map<String, Boolean> updatedUserFlagsMap(
       Map<String, Object> userMap, Map<String, Object> userDbRecord) {
     Map<String, Boolean> userBooleanMap = new HashMap<>();
+    if(userDbRecord.get(JsonKey.EMAIL) != null && (userDbRecord.get(JsonKey.EMAIL_VERIFIED) == null
```
Comment (manzarul) [inline actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java]: convert into a separate method

Comment (Hari-stackroute) [reply actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java]: Done

**Pull request merged**